### PR TITLE
UI refresh

### PR DESCRIPTION
**Pull request by arodebaugh**
This is a full UI refresh. There will be future cards and PRs to make this better. But this is the barebones start of it

image

### CODE REVIEW

File: src/rwv/ui/plot_widget.py [hunk 1]
```diff
@@ -75,52 +84,94 @@ def __init__(self, window, db):
...
 
+        self.create_canvas_window()
+
+    def create_canvas_window(self):
+        """
+        Creates window that displays the generate chart.
+        """
+        self.canvas_window = QtWidgets.QWidget()
+        self.canvas_window.setWindowTitle("Endurance")
+        self.canvas_window.closeEvent = self.close_application
```
Comment (yuli091280) [inline src/rwv/ui/plot_widget.py:109]:
```suggestion
        self.canvas_window.closeEvent = lambda: self.close_application
```

File: src/rwv/ui/plot_widget.py [hunk 2]
```diff
@@ -75,52 +84,94 @@ def __init__(self, window, db):
...
+        menu_bar.addMenu(file_menu)
+
+        # Action to close the database file.
+        close_current_db = file_menu.addAction("Close Current DB")
+        close_current_db.triggered.connect(lambda: self.window.reset())
+
+        # Action to save the graph.
+        save_graph = file_menu.addAction("Save Graph")
+        save_graph.triggered.connect(lambda: self.save_current_graph())
+        save_graph.setShortcut('Ctrl+S')
```
Comment (yuli091280) [inline src/rwv/ui/plot_widget.py:145]: If I was a user I'd expect to ctrl+S to save the plot when the plot is the active window. However right now the shortcut only works when the control window is the active window

**Review by yuli091280 — CHANGES_REQUESTED** [review]
Now that the plot and the controls are in 2 different windows, they should also be 2 different widgets in 2 different files.
Also note that you currently have a bug where closing the db will not close off the previous plot window. If the user then open a new db they will get another plot window. Closing the old plot window will result in the whole thing getting closed. You need to change reset to make sure the old window gets destroyed.

File: src/rwv/ui/plot_widget.py (hunk 2 above)
Comment (arodebaugh) [reply src/rwv/ui/plot_widget.py:145]: This issue seems to be Windows/Linux specific. Will work with @mtp75 to hopefully get this fixed

Comment (arodebaugh) [reply src/rwv/ui/plot_widget.py:144]: This along with the UI issue for Windows/Linux will be fixed in a PR that will be created a little later.

Comment (yuli091280) [reply src/rwv/ui/plot_widget.py:144]: plz make a ticket for the fix u'll be working on

**Review by yuli091280 — APPROVED** [review]
Other than what was already mentioned, looks good.

**Pull request merged**